Bottom: b89740b4f4586e57480cf02efb852da8544aa0bb
Top:    b7698b5e26c88bf735a9ce997fae5fe068fe6b62
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-07 15:20:04 -0500

Remove local interface configuration

exim4 container will listen on all interfaces available within
the container.


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index c6290553..e6fb458d 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -28,9 +28,6 @@ case ${1} in
             [ -e /etc/mailname ] \
                 && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
-            # set local IP addr
-            LOCAL_IP_ADDR=$(grep "[ \t]*${HOSTNAME}[ \t]*" /etc/hosts|cut -f 1)
-            sed -i 's|dc_local_interfaces=.*|dc_local_interfaces='"'${LOCAL_IP_ADDR}'"'|' ${CONFIG_PATH}/update-exim4.conf.conf
             # copy certificate
             [ -e /etc/ssl/private/exim.crt ] \
                 && rm -f ${CONFIG_PATH}/exim.crt \
